scripts/start

```diff
@@ -1,23 +1,27 @@
 #!/bin/bash
 
-cd $(dirname $0)/..
-
-exists() {
-    [ -e "$1" ]
-}
-
-if exists Artemis-modules-extra; then
-    ADDITIONAL_OPTIONS=--scale=karton-sqlmap=5
+if [[ $(uname -m) != "arm"* ]]; then
```

Comment:
too big duplicate - instead of copying the whole run_docker_compose command you can write it only once and add --profile $PROFILE, only changing the variable